python/pyspark/sql/tests/test_functions.py

```diff
@@ -197,6 +197,18 @@ def test_string_functions(self):
                 df.select(getattr(functions, name)("name")).first()[0],
                 df.select(getattr(functions, name)(col("name"))).first()[0])
 
+    def test_octet_length_function(self):
+        from pyspark.sql.functions import octet_length
+        df = self.spark.createDataFrame([('cat',), ('\U0001F408',)], ['cat'])
+        actual = df.select(octet_length('cat')).collect()
+        self.assertEqual([Row(3), Row(4)], actual)
+
+    def test_bit_length_function(self):
+        from pyspark.sql.functions import bit_length
```

Comment:
I know this is sort of consistent with other tests but in fact it's totally fine to import it on the top. Since this is a new feature that's not going to be backported, let's move this import to the top.